### PR TITLE
Adds very simple form to the home page

### PR DESCRIPTION
**Pull request by mmurray22**
Adds a simple form to the home page. Here is the screenshot of the website: 
image

### CODE REVIEW

File: capstone/src/main/webapp/index.jsp [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <html>
```
Comment (ccondit) [inline capstone/src/main/webapp/index.jsp:1]: nit - add `<!DOCTYPE html>` at the start of this document.

File: capstone/src/main/webapp/index.jsp [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-<body>
-<h2>Hello World!</h2>
-</body>
+  <body>
+    <h2>Welcome to UniViz!</h2>
```
Comment (ccondit) [inline capstone/src/main/webapp/index.jsp:3]: also I think you [also want to add](https://google.github.io/styleguide/htmlcssguide.html#Encoding):
`<meta charset="utf-8">`
in a `head` element

File: capstone/src/main/webapp/index.jsp (hunk 1 above)
Comment (mmurray22) [reply capstone/src/main/webapp/index.jsp:1]: done!

File: capstone/src/main/webapp/index.jsp (hunk 2 above)
Comment (mmurray22) [reply capstone/src/main/webapp/index.jsp:7]: done!

File: capstone/src/main/webapp/search.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function autocomplete(data) {
```
Comment (ccondit) [inline capstone/src/main/webapp/search.js:1]: Looks like this got added into this PR after the fact. Can you send this as a separate PR?

File: capstone/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
 <html>
-<body>
-<h2>Hello World!</h2>
-</body>
+  <head>
+  <meta charset="utf-8">
+  </head>
+  <body>
+    <h2>Welcome to UniViz!</h2>
+      <form action="/" method="POST" autocomplete="off">
```
Comment (ccondit) [inline capstone/src/main/webapp/index.jsp:8]: nit - consider leaving the action and method off of this form until they're needed.

File: capstone/src/main/webapp/search.js (hunk 3 above)
Comment (mmurray22) [reply capstone/src/main/webapp/search.js]: yep! just reverted the commit!

File: capstone/src/main/webapp/index.jsp (hunk 4 above)
Comment (mmurray22) [reply capstone/src/main/webapp/index.jsp]: addressed!

**Comment by mmurray22** [comment]
Does the spacing in the HTML file look weird to any of you? I've tried fixing it both in Github UI, CloudShell Editor, and the Cloudshell Vim and it's messed up in all locations and I can't seem to fixed it...

**Comment by ccondit** [comment]
> Does the spacing in the HTML file look weird to any of you? I've tried fixing it both in Github UI, CloudShell Editor, and the Cloudshell Vim and it's messed up in all locations and I can't seem to fixed it...

There's a mixed tab / space issue - the last `input` element should use spaces instead of tabs

image

**Comment by mmurray22** [comment]
Finally fixed!

**Comment by mmurray22** [comment]
Just changed!
